bin/summarize_resistance_mixed_infection.py

```diff
@@ -136,7 +136,7 @@ def create_resistance_df(sample_res, method):
 #===============
     stats_df = pd.read_csv(args["merged_cohort_stats_file"], sep="\t")
     mixed_inf_stats_df = stats_df[stats_df["RELABUNDANCE_THRESHOLD_MET"] == 0]
-    filtered_stats_df = mixed_inf_stats_df[(mixed_inf_stats_df[["MAPPED_NTM_FRACTION_16S_THRESHOLD_MET", "COVERAGE_THRESHOLD_MET", "BREADTH_OF_COVERAGE_THRESHOLD_MET"]] == 1).all(axis=1)]
+    filtered_stats_df = mixed_inf_stats_df.loc[ (mixed_inf_stats_df["RELABUNDANCE_THRESHOLD_MET"]==0) & (mixed_inf_stats_df["MAPPED_NTM_FRACTION_16S_THRESHOLD_MET"]==1) & (mixed_inf_stats_df["COVERAGE_THRESHOLD_MET"]==1) & (mixed_inf_stats_df["BREADTH_OF_COVERAGE_THRESHOLD_MET"]==1)]
```

Comment:
@vrennie @mdediegofuertes , the issue was hidden in this refactor here regarding the NTM samples not showing up.